java/lang/String/String.cpp

```diff
@@ -845,4 +850,170 @@ boolean String::operator>=(const String &target) const {
 		return false;
 	}
 	return true;
+}
+
+String String::print(const String &format, short value)
+{

```

Comment:
{ should almost always be at the end of the previous line. We follow the google code style for C++.